app/controllers/CalculationController.scala

```diff
@@ -47,13 +47,16 @@ class CalculationController @Inject()(authenticate: AuthenticationPredicate,
 
   val action: ActionBuilder[MtdItUser] = checkSessionTimeout andThen authenticate andThen retrieveNino andThen retrieveIncomeSources
 
-  private def view(taxYear: Int, calculation: Calculation, transaction: Option[TransactionModel] = None, incomeBreakdown: Boolean = false)(implicit request: Request[_]): Html = {
+  private def view(taxYear: Int, calculation: Calculation, transaction: Option[TransactionModel] = None,
```

Comment:
Can you put all parameters on new lines, to keep it consistant?

Reply:
Done